tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -340,6 +344,9 @@ void ROOT::Experimental::RNTupleWriter::CommitCluster(bool commitClusterGroup)
          CommitClusterGroup();
       return;
    }
+   if (fSink->GetWriteOptions().GetHasSmallClusters() && (fUnzippedClusterSize > 512 * 1024 * 1024)) {
```

Comment:
I think it would be clearer if we define a constant instead, e.g. `kMaxSmallClusterSize`.